client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -426,7 +426,13 @@ const connectComponent = connect( ( state, ownProps ) => {
 	const selectedCategory = categories && head( filter( categories, { ID: podcastingCategoryId } ) );
 	const podcastingFeedUrl = selectedCategory && selectedCategory.feed_url;
 
-	const isCategoryChanging = podcastingCategoryId !== ownProps.settings.podcasting_category_id;
+	const isSavingSettings = isSavingSiteSettings( state, siteId );
+	const isCategoryChanging =
+		! isSavingSettings &&
+		! ownProps.isRequestingSettings &&
+		ownProps.settings &&
+		Number( ownProps.settings.podcasting_category_id ) > 0 &&
+		podcastingCategoryId !== Number( ownProps.settings.podcasting_category_id );
```

Comment:
Not necessary for this PR, but is there a single place we could take care of calling `Number` on `podcasting_category_id` so that code using it doesn't have to concern itself with this?